mongodbforms/fieldgenerator.py

```diff
@@ -130,10 +130,7 @@ def get_field_label(self, field):
         return ''
 
     def get_field_help_text(self, field):
-        if field.help_text:
-            return field.help_text
-        else:
-            return ''
+        return getattr(field, 'help_text', '')
```

Comment:
This code is not equal.

Reply:
fd183d0

Reply:
506e284e2fdc5e5d0777f987879bab3ae53be075